README.md

```diff
@@ -4,4 +4,8 @@
 
 Please email Dave (`dpane@cmu.edu`) (for all issues), cc'ing `help@cs.cmu.edu` (don't cc'ing if you are responsible for the issue. THIS IS IMPORTANT).
 
+Don't know whether you should be responsible for the issue? 
+Please review the twiki (http://twiki.cnbc.cmu.edu ) about the policies, specifically:
```

Comment:
change `(http://twiki.cnbc.cmu.edu )` to `(<http://twiki.cnbc.cmu.edu>)` so that people can click on it directly. Same for next line.